robot_controllers/include/robot_controllers/cartesian_twist.h

```diff
@@ -148,7 +152,9 @@ class CartesianTwistController : public Controller
 
   boost::mutex mutex_;
   KDL::Twist twist_command_;
+  std::string twist_command_frame_;
   ros::Time last_command_time_;
+  bool is_active_;
```

Comment:
walking through the logic -- it doesn't look like we even need an "is_active_" flag -- if we don't get commands for 0.5 seconds or get an empty command, we call requestStop(), at which point things don't get called anymore